<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #000000; /* Ð¡Ð¸Ð½Ð¸Ð¹ Ñ†Ð²ÐµÑ‚ */
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="chatDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>781</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: black;
    border: 1px solid #05fbff;
    border-radius: 10%;
    color: black;
    font-size: 14px;
}
</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>781</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 1px solid #555;
    border-radius: 10px;
    background-color: black;
    text-align: center;
color: whitesmoke;
}

QProgressBar::chunk {
background-color: grey;
    border-radius: 5px;
}
</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 24px;
    font-weight: bold;
    color: #fff;
    background-color: #333;
    padding: 10px;
    border-radius: 10px;
    box-shadow: 0 0 10px rgba(0, 0, 0, 0.5),
                 0 0 20px rgba(0, 0, 0, 0.2) inset;
}
</string>
    </property>
    <property name="text">
     <string>Anon.Net</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>471</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
color: white;
}

</string>
    </property>
    <property name="text">
     <string>{smth} is writing...</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: #1E1E1E;
    color: purple;
    border: 1px solid grey;
    border-radius: 8px;
    padding: 5px 10px;
    font-size: 10px;
    font-family: 'Arial', sans-serif;
    transition: background-color 0.3s, border-color 0.3s;
}

QComboBox:hover {
    background-color: #2C2C2C;
}

QComboBox::drop-down {
    background-color: #1E1E1E;
    border-radius: 8px;
    padding: 5px;
}

QComboBox::indicator {
    width: 15px;
    height: 15px;
}

QComboBox QAbstractItemView {
    background-color: #2E2E2E;
    color: #FFFFFF;
    selection-background-color: #4CAF50;
    selection-color: #FFFFFF;
    border: 1px solid #4CAF50;
    border-radius: 8px;
}

QComboBox QAbstractItemView::item {
    padding: 10px;
}

QComboBox QAbstractItemView::item:selected {
    background-color: #4CAF50;
    color: #FFFFFF;
}
</string>
    </property>
    <item>
     <property name="text">
      <string>France</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>England</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>USA</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px dashed aqua;</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">QWidget {
color: white;

}
</string>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">QWidget {
color: blue;

}
</string>
         </property>
         <property name="text">
          <string>289fh3u</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">QWidget {
color: white;

}
</string>
         </property>
         <property name="text">
          <string>IP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">QWidget {
color: red;

}
</string>
         </property>
         <property name="text">
          <string>1.1.1.1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="styleSheet">
          <string notr="true">QWidget {
color: white;

}
</string>
         </property>
         <property name="text">
          <string>region-time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="styleSheet">
          <string notr="true">QWidget {
color: green;

}
</string>
         </property>
         <property name="text">
          <string>12:30</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>476</y>
      <width>43</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: black;
    border: 1px solid white;
    border-radius: 10%;
    color: #05fbff;
    font-size: 13px;
text-align: center;
padding: -1px;
}
</string>
       </property>
       <property name="text">
        <string> ðŸ—Ž</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: black;
    border: 1px solid white;
    border-radius: 10%;
    color: #05fbff;
    font-size: 12px;
text-align: center;
padding: 0px;
}
</string>
       </property>
       <property name="text">
        <string>#</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>761</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: black;
    border: 1px solid lime;
    border-radius: 10%;
    color: black;
    font-size: 14px;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="messageInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: black;
    border: 1px solid #05fbff;
    border-radius: 10%;
    color: white;
    font-size: 14px;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>510</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: black;
    border: 1px solid white;
    border-radius: 10%;
    color: #05fbff;
    font-size: 14px;
}
</string>
    </property>
    <property name="text">
     <string>Send...</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>741</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollArea {
border: 1px solid black;
}</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>739</width>
       <height>99</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>761</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid pink;
    border-radius: 8px;
    margin-top: 20px;
    background-color: black;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 5px;
    font-weight: bold;
    font-size: 14px;
    color: #4A90E2;
}
</string>
    </property>
    <property name="title">
     <string>PinðŸ“Œ</string>
    </property>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>741</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollArea {
border: 1px solid black;
}</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>739</width>
        <height>19</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
